src/app/about/page.tsx

```diff
@@ -29,11 +33,11 @@ export default function About() {
           <div>
             <p>
               Thank you for stopping by my online corner. Feel free to check out
-              the handmade jewelry and clothing peaces that are ready and
+              the handmade jewelry and clothing pieces that are ready and
               waiting for buying.
             </p>
             <div className="py-5 text-center md:text-left">
-              <LinkBtn title="Welcome to Shop" pathName="/shop" />
+              <LinkButton title="Welcome to Shop" pathName="/shop" />
```

Comment:
You should design your component to be uses like this:
```suggestion
  <LinkButton href="/shop">Welcome to Shop</LinkButton>
```

So then you'll be able to easily pass complex children types. And it'll feel more native then